README.md

```diff
@@ -5,45 +5,61 @@ Sn4k3 - next generation MMO
 
 ![Frendly snek demo](sn4k3.gif)
 
-# Installation
+# Requirements <small>(if you don't have Docker)</small>
 
-Install dependencies
+* PHP >=7.0
+* Python >=2.7 (with `pip`)
+* NodeJS >=4.4 (with `npm`)
 
-```bash
-# Root install of crossbar
-pip install crossbar
+# Install
+
+## With Docker
+
+Just run `docker-compose build`.
 
-# Backend dependencies
-cd backend/
-composer install
+## Without Docker
 
-# Frontend dependencies
-cd frontend/
-npm install
+Execute all these commands directly from the root directory of the repository.
+
+```bash
+pip install crossbar
+composer --working-dir=backend install
+npm --prefix=frontend install
 npm run build
-# In dev, you can run this:
-npm run watch
 ```
 
-Configure your web-server at `frontend/public`
+# Run
+
+## With Docker
 
-:information_source: If you want to install it with docker, please follow this link:<br>
-http://crossbar.io/docs/Installation-on-Docker/
+Just execute `docker-compose up -d`
 
-# Execute
+And go to `127.0.0.1:8080`.
+
+## Without docker
+
+You will need at least 3 terminals for this, because all apps need to be run independently.
 
 Run Crossbar middleware instance
 
 ```bash
-crossbar start
+crossbar start --config=crossbar/config.json
```

Comment:
Nope, I moved the crossbar config because we needed it for the specific Dockerfile. This is why here I explicitly ask for configuration file to allow using the same config file as Docker (because now docker "becomes the reference", it's the goal)